### PR TITLE
index.html

### PR DESCRIPTION
**Pull request by leyadotnet**
First Web Page Using Brackets

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                </ul>
+        <h2><a href="http://www.myrecipes.com/recipe/chicken-rice-parmesan-skillet">Lunch Recipe</a></h2>
+                <ul>
+                    <li>Lasagna</li>
+                </ul>
+    </main>
+    <footer>
+        <small><i>Copyright &copy; 2016 Leya</i></small>
+      <ul>
+          <li>E-mail: <a href="mailto:Firstname@yourlastname.com">Firstname@yourlastname.com</a></li>
```
Comment (mbMosman) [inline index.html]: You might stick some real-looking text in the email mailto, so it looks less generic on the page.  Just an appearance thing.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                <ul>
+                    <li>cereal</li>
+                </ul>
+        <h2><a href="http://www.myrecipes.com/recipe/chicken-rice-parmesan-skillet">Lunch Recipe</a></h2>
+                <ul>
+                    <li>Lasagna</li>
+                </ul>
+    </main>
+    <footer>
+        <small><i>Copyright &copy; 2016 Leya</i></small>
```
Comment (mbMosman) [inline index.html]: You should pull out the small and italic tags for now.  It's not good "semantic" html, even though they show this in the book.  In another week you'll know how to do it w/ CSS.

**Comment by mbMosman** [comment]
Nicely done... leaving open for next week

**Comment by leyadotnet** [comment]
Thanks Mary!

**Comment by leyadotnet** [comment]
Having all styles in an external CSS file and out of the HTML files is good for better maintenance and good to have all style rules in a separate place than html and clean html code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Food Blog</title>
+        <meta charset="utf-8" />
+        <link rel="stylesheet"  href="style.css">
+    </head>
+<body>
+    <main>
+        <header>
```
Comment (mbMosman) [inline index.html]: Usually the header will be outside of the `main` element.

**Comment by mbMosman** [comment]
Nice work for both part 1 & 2.

**Comment by leyadotnet** [comment]
Thank you! :)

**Comment by mbMosman** [comment]
I'm noticing for part 3 that your image is not displaying.  The web server is case sensitive, so you need to have the img src match exactly the file path.  You have lowercase in the scr, but the directory is capitalized.

If you fix this, please open a new pull request for this week.

**Pull request closed without merging**